server/views/legal-responsibility.html

```diff
@@ -0,0 +1,45 @@
+{% extends 'form-layout.html' %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+
+{% block formContent %}
+  <div class="flash-card">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full"
+
+        <header>
+          <h1 class="govuk-heading-l" id="pageTitle">{{ pageTitle }}</h1>
+        </header>
+
+      </div>
+    </div>
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        {% for helpTextPara in helpTextParas %}
+          <p class="govuk-body" id="helpTextPara-{{ loop.index }}">{{ helpTextPara }}</p>
+        {% endfor %}
+
+        <div class="govuk-inset-text" style="border-left: 10px solid white">
+          <p id="callOutText">{{ callOutText }}</p>
+        </div>
+
+        {{ govukButton({
+            attributes: {
+              id: "continue"
+            },
+            html: '<div style="color:#0b0c0c">Continue</div>',
```

Comment:
Avoid inline style, create a class using GOV UK colour of  $govuk-text-colour. Probably wouldn't need to use html attribute or the DIV, could probably just use text: '....' instead